ui/src/main/webapp/home/index.js

```diff
@@ -113,31 +66,36 @@ const LdapTileView = (props) => {
     baseUserDn,
     onDeleteConfig,
     ldapUseCase,
-    attributeMappings
+    attributeMappings,
+    submitting
   } = props
 
   return (
-    <Paper className={styles.config}>
-      <div className={styles.tileTitle}>{ldapUseCase === 'authentication' ? 'LDAP Authentication Source' : 'LDAP Attribute Store'}</div>
-      <ConfigField fieldName='Hostname' value={hostName} />
-      <ConfigField fieldName='Port' value={port} />
-      <ConfigField fieldName='Encryption Method' value={encryptionMethod} />
-      <ConfigField fieldName='Bind User' value={bindUser} />
-      <ConfigField fieldName='Bind User Password' value='******' />
-      <ConfigField fieldName='UserName Attribute' value={userNameAttribute} />
-      <ConfigField fieldName='Base Group DN' value={baseGroupDn} />
-      <ConfigField fieldName='Base User DN' value={baseUserDn} />
-      { ldapUseCase !== 'authentication'
-        ? (
-          <MapDisplay label='Attribute Mappings'
-            mapping={attributeMappings} />
-      ) : null }
-      <RaisedButton style={{marginTop: 20}} label='Delete' secondary onClick={onDeleteConfig} />
+    <Paper className={styles.outerSpinner}>
+      <Spinner submitting={submitting}>
+        <div className={styles.innerSpinner}>
```

Comment:
Definitely something we should discuss more outside this PR and possibly make a ticket to address it.